Source/bmalloc/libpas/src/libpas/pas_utils.h

```diff
@@ -1112,13 +1122,13 @@ static inline unsigned pas_hash_ptr(const void* ptr)
 /* Undefined for value == 0. */
 static inline unsigned pas_log2(uintptr_t value)
 {
-    return (sizeof(uintptr_t) * 8 - 1) - (unsigned)__builtin_clzl(value);
```

Comment:
We should have `PAS_TESTING_ASSERT(sizeof(uintptr_t) == sizeof(unsigned long long))`. Otherwise, this is wrong.